app/models/service.rb

```diff
@@ -43,7 +43,7 @@ class Service < ApplicationRecord
   end
 
   def self.columns
-    super.reject { |column| column.name == 'buyer_can_see_log_requests'}
+    super.reject { |column| %w[buyer_can_see_log_requests tech_support_email admin_support_email].include?(column.name) }
```

Comment:
@Martouta What does this do exactly? I thought it rejected updates but the test was failing, I had to add `tech_support_email`and `admin_support_email` to `attr_readonly`.